common/geth/handle_error.go

```diff
@@ -0,0 +1,88 @@
+package geth
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/txpool"
+)
+
+type ChainConnFaults int64
+
+const (
+	SenderFault ChainConnFaults = iota
+	RPCFault
+	TooManyRequest
+	Ok
+	EVMFault
+)
+
+// this function accepts error message returned from API
```

Comment:
nit: `// HandleError accepts error message...`
Comments should start with the name of the method, variable, etc.